Scripts/models/logit.py

```diff
@@ -426,10 +428,37 @@ def calc_individual_mode_prob(self, is_car_user, zone):
                     else:
                         mode_exps[mode] *= math.exp(b["car_users"][1])
             mode_expsum += mode_exps[mode]
+            if mode == "car":
+                car_expsum += mode_exps[mode]
+            else:
+                sust_expsum += mode_exps[mode]
         probs = numpy.empty(len(modes))
         for i, mode in enumerate(modes):
             probs[i] = mode_exps[mode] / mode_expsum
-        return probs
+        # utils to money
+        logsum = numpy.log(mode_expsum)
+        sust_logsum = numpy.log(sust_expsum)
+        car_logsum = numpy.log(car_expsum)
+        try:
+            b = self.dest_choice_param["car"]["impedance"]["cost"]
+        except KeyError:
+            # School tours do not have a constant cost parameter
+            # Use value of time conversion from CBA guidelines instead
+            b = -0.31690253
```

Comment:
This code is now used in two places. Maybe we should put it in a separate function?